ui/package.json

```diff
@@ -29,6 +29,6 @@
     "vue-template-compiler": "^2.6.6"
   },
   "peerDependencies": {
-    "vue": "^2.6.6"
+    "vue": ">=2.6"
```

Comment:
hats (`^`) are preferred.

If this is your desired value, please write as `^2.6.0`

Reply:
You are right. Don't know why I switched it to `>=` to be honest. Good catch. Thanks.